### ISSUE TITLE
SignIn Ineffective Alternative Text 

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#787]
The logo image included on this page (with the filename “logo-black.png”) only has the alternative text of “logo”.  This does not adequately describe its purpose or content to screen reader users. 
 
Suggested Solution 
 
Modify the text of the alt attribute on the < img> to describe the purpose of the image.  It’s pointless for screen reader users to encounter a logo if the alt text doesn’t even indicate whose logo it is, so that information should be included as a minimum.

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/77c838208a1fe397f69c68d5653a500f6d496d82
